aptos-move/framework/aptos-stdlib/sources/simple_map.spec.move

```diff
@@ -56,12 +56,29 @@ spec aptos_std::simple_map {
         pragma verify=false;
     }
 
-    spec to_vec_pair {
-        pragma verify=false;
+    spec to_vec_pair<Key: store, Value: store>(map: SimpleMap<Key, Value>): (vector<Key>, vector<Value>) {
+        pragma intrinsic;
+        pragma opaque;
```

Comment:
Why do we use both `opaque` and `intrinsic`? I am not sure how they would work together because I haven't used both at the same time in a function.

Reply:
If `intrinsic` is not specified, a boogie procedure will be generated for `to_vec_pair` and an error will be raised because it accesses the `data` field of `SimpleMap`. 
If `opaque` is not specified,  the code in `to_vec_pair` will be inlined, which causes the same error.